content/blog/summary-from-the-9th-annual-chapel-implementers-and-users-workshop-chiuw-2022.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Summary from The 9th Annual Chapel Implementers and Users Workshop (CHIUW 2022)
+date: 2022-07-15T19:31:06.645Z
+priority: ""
+author: Michelle Strout and Engin Kayraklioglu
+authorimage: /img/mstrout-headshot-small.jpg
+thumbnailimage: /img/engin.png
+tags:
+  - Chapel Programming Language
+  - GPU Programming
+  - High Performance Computing
+  - Open Source
+  - Arkouda (Data Analytics in Python powered by Chapel)
+---
+**Introduction**
+
+Programming today is complicated by the many kinds of parallelism that exist in everything from phones to laptops to supercomputers.  The open-source Chapel parallel programming language is being developed at HPE with the goals of making programming more productive / easier, while still enabling high performance that takes advantage of the wide variety of parallelism available today.
+
+In this post, we summarize highlights from the recent [9th Annual Chapel Implementers and Users Workshop (CHIUW 2022)](https://chapel-lang.org/CHIUW2022.html).  We are the general chair of the workshop, Michelle Strout, and the program committee chair, Engin Kayraklioglu, and members of the Chapel development team at HPE.  Read on to hear about some exciting applications that are using Chapel productively, the coding day that happened the day before the workshop, updates on the project, and feedback the team received during and after the workshop.
+
+**Applications Written in Chapel**
+
+Programmers generally enjoy trying out new programming languages, but before they lean on a programming language it is always better to see example use cases.  The Chapel programming language is being used productively in a range of application domains: data science, aeronautical simulations, cosmology simulations, and quantum diagonalization to name a few. CHIUW featured the following talks on different Chapel applications:
+
+* Large-Scale and User-Friendly Exact Diagonalization in Chapel (<https://youtu.be/vBxPTzIRRr0>)
+* Recent Developments in the CHApel Multi-Physics Simulation Software (<https://youtu.be/uTE_RZkODOk>)
+* UltraLight Dark Matter in Simulations: A Chapel-Powered Eigenstate Perspective (<https://youtu.be/YrXYpgnt4rQ>)
+* Implementing and Optimizing Parquet I/O in Chapel (<https://youtu.be/pstRsgMhCDA>)
+* Truss Analytics Algorithms and Integration in Arkouda (<https://youtu.be/xI9EByv7A5M>)
+* From C and Python to Chapel as My Main Programming Language (<https://youtu.be/400jmMzdzHQ>)
+
+**Coding Day**
+
+For coding day, anyone interested in working one-on-one or in small groups with developers from the Chapel team at HPE could indicate their interest in an online form.  We had 7 different sessions that occurred.  Programmers interested in Chapel were able to ask questions specific to their Chapel code, interactively make changes to the code, and work through issues with Chapel developers around for immediate assistance. In these sessions, Chapel users and developers worked on
```

Comment:
Take out "that occurred" in the second sentence so it just reads: "We had 7 different sessions." Or, if you want to keep it, add a little more. Something like "We had 7 different sessions occuring simultaneously." or something like that.

In the second to last sentence, it should be worded: " ... work through issues with Chapel developers who were present to provide immediate assistance."

Place a colon after "worked on:"